nexus-writer/diagrams/.gitignore

```diff
@@ -0,0 +1 @@
+!*.svg
```

Comment:
Is there a need to have SVG in the root `.gitignore`?

Reply:
No, that folder is a duplicate of docs in fact

Reply:
You will now need to alter the root `.gitignore`.